### ISSUE TITLE
Spotify support 

### ISSUE THREAD

**Issue by martenumberto** [alexa/alexa-avs-sample-app#968]
Are there any news of supporting Spotify music playback?

**Comment by dhpp** [comment]
Thanks for asking about this - this is on our feature roadmap, but we can't comment on schedule.

**Comment by jackwhitehead** [comment]
@martenumberto I have figured a way to control spotify from alexa sample app, but it requires a fair knowledge of computers. I can help if you need though

**Comment by ekennedy80** [comment]
@jackwhitehead I am a software engineer and was wondering if you could explain to me what your solution is to controlling Spotify from the Alexa sample app.

**Comment by jackwhitehead** [comment]
@ekennedy80 sure thing! would you like me to email you the documentation?

**Comment by dionator** [comment]
@ekennedy80 I'm seriously interested in your documentation :)
Can you share it? if saw how do I send you my email without revelling it to everyone? (not expert on GitHub yet)

**Comment by ekennedy80** [comment]
@jackwhitehead Sorry for responding so late, work has been very busy.  I would really appreciate if you could email me your documentation.  Maybe we can collaborate?

**Comment by kekkokk** [comment]
Hi guys!

any news on that?

**Comment by PaulSchreiber96** [comment]
@jackwhitehead , @ekennedy80 
Hey guys, i read that you have a workaround for spotify from the alexa sample app.
Could you mail me the documentation?

**Comment by AndreVizosodaCruz** [comment]
Hello,
Any update so i can control spotify from alexa ?
thanks

**Comment by polgafr** [comment]
> @martenumberto I have figured a way to control spotify from alexa sample app, but it requires a fair knowledge of computers. I can help if you need though

Hi, I just installed Alexa in my Raspberry B and all the functionalities are good, but Spotify still can't be used. Neither Amazon Music.
Maybe you could pass me your documentation if you think it's fine.

Thanks.

**Comment by jackwhitehead** [comment]
no problem. just completing exams this week coming, but I will be sure to write proper documentation. Please be aware that this is not a perfect and permanent solution, as it is completely free. However, there are low cost permanent alternatives

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by polgafr** [comment]
Yeah, no problem. I'm with exams too so...
I have in mind that that's not a permanent solution. The other "low cost" alternaties, you mean are an Amazon Echo I imagine.

**Comment by jackwhitehead** [comment]
not necessarily. you might ne familiar with pagekite, which is a permanent DNS forwarding service. In this I use the free alternative ngrok. I would advise you to familiarise yourself with DNS forwarding services should you decide to setup a permanent solution

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by polgafr** [comment]
Okay. I'll look for information about that. And I wait for your documentation. When you can. Thanks!
